common/persistence/src/main/resources/db/migration/V3__createTablesForDownload.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE federation_batch_info (
+    batch_tag   varchar(20) PRIMARY KEY,
```

Comment:
[EFGS uses varchar(64) for their batch fields](https://github.com/eu-federation-gateway-service/efgs-federation-gateway/blob/master/src/main/resources/db/changelog/v000-create-diagnosis-key-batch-entity-table.yml#L21-L26). Would be good to match that, just in case they start sending longer tags with hashed values or something similar.

Reply:
The specification is stating an even smaller value. (not sure how reliable the specification is though)